strings/strings.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Return a number comparing two strings lexicographically.
+ *
+ * @example
+ * ```ts
+ * import { compare } from "https://deno.land/std@$STD_VERSION/strings/strings.ts";
+ *
+ * compare("a", "b"); // -1
+ * compare("a", "a"); // 0
+ * compare("b", "a"); // 1
+ * ```
+ */
+
+export function compare(a: string, b: string): number {
+  if (a === b) {
+    return 0;
+  } else if (a < b) {
+    return -1;
+  } else {
+    return 1;
+  }
+}
+
+/**
+ * Report whether a substring is within a string.
+ *
+ * @example
+ * ```ts
+ * import { contains } from "https://deno.land/std@$STD_VERSION/strings/strings.ts";
+ *
+ * contains("Hello, Deno!", "Deno"); // true
+ * ```
+ */
+
+export function contains(s: string, substr: string): boolean {
+  return s.includes(substr);
+}
```

Comment:
I don't like this, people should be using the native function in this case.

At _worst_ this function show throw a NotSupportedError instead and have an `@obsolete` attribute which says please use `s.includes(substr)` which would help people who gravitate to the word "contains" to be redirected to the right place which is the built-in function called `includes`.

But I would not just wrap standard functions and have people write a bunch of code that differs from the rest of the community.